docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+version: "3.2"
+
+services:
+
+  functions:
+    image: functions-template:v2.0.0
+    build:
+      context: ./
+      dockerfile: docker/functions/Dockerfile
+    env_file: 
+      - .env
+    environment:
+      - languageWorkers__node__arguments=--inspect=0.0.0.0:5858
+    working_dir: /usr/src/app
+    ports:
+      - "7071:7071"
+      - "5858:5858"
+    networks:
+      - io-fn
+    depends_on:
+      - fnstorage
+    volumes:
+      - .:/usr/src/app
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.functions.rule=Host(`localhost`)"
+      - "traefik.http.routers.functions.entrypoints=web"
+      - "traefik.http.services.functions.loadbalancer.server.port=7071"
+
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-id=unused"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-groups=${REQ_USER_GROUPS}"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-subscription-id=${REQ_SERVICE_ID}"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-email=${SERVICE_EMAIL}"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-note=unused"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-functions-key=unused"
+
+      # apply middleware to route
+      - "traefik.http.routers.functions.middlewares=testHeader"
+
+  fnstorage:
+    image: azurite
+    build:
+      context: ./
+      dockerfile: docker/azurite/Dockerfile
+    ports:
+      - "10000:10000"
+      - "10001:10001"
+      - "10002:10002"
+    networks:
+      - io-fn
+
+  storage:
+    image: azurite
+    command: ["sh", "-c", "node bin/azurite -l /opt/azurite/folder --blobPort 10003 --queuePort 10004 --tablePort 10005"]
+    ports:
+      - "10003:10003"
+      - "10004:10004"
+      - "10005:10005"
+    depends_on:
+      - fnstorage
+    networks:
+      - io-fn
+
+  cosmosdb:
```

Comment:
We won't use cosmos, we can remove the service

Reply:
Fix with 066daa4